packages/components/header/_header.scss

```diff
@@ -669,22 +719,6 @@
 
   .nhsuk-header__transactional-service-name {
     float: left;
-    padding-left: $nhsuk-header-spacing;
```

Comment:
`$nhsuk-header-spacing` is no longer used after removing it, can we change the value and reuse?